CHANGELOG.md

```diff
@@ -25,6 +25,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.30.1] - 2024-08-13
+
+### Fixed
+- Fixed a traceback error by Adding exception to handle InvalidGrant Error.This 
+  would address the traceback errors shown during sat authentication while 
+  executing other sat commands as well.
```

Comment:
Probably no need to mention the specific exception class. Other wording suggestions:

```suggestion
- When the token used to access the API gateway is expired, log a helpful error
  message instructing the user to reauthenticate with `sat auth` and then exit
  instead of printing a traceback and exiting.
```